packages/gestalt/src/Box.js

```diff
@@ -307,6 +309,14 @@ const lgPaddingX = bind(rangeWithoutZero('lgPaddingX'), whitespace);
 const lgPaddingY = bind(rangeWithoutZero('lgPaddingY'), whitespace);
 const lgPadding = union(lgPaddingX, lgPaddingY);
 
+const opacityMap = mapClassName(name => styles[name]);
+const opacity = val => {
+  if (val > 0 && val < 1) {
```

Comment:
@jennyscript Can we avoid this fork?

`String(val).replace('.', '')`

```
[0, 0.3, 1].map(val => String(val).replace('.', ''))
// ["0", "03", "1"]
```

Reply:
Range only takes number arguments. I'll see if one of the string compatible ones could use that form

Reply:
I tried a couple hacky workarounds, but this is the most clear option so I'd prefer to keep it